src/views/download/download.scss

```diff
@@ -49,23 +49,34 @@ $developer-spot: $ui-yellow;
             }
         }
     }
-    .inner {
+    
+    .download-content {
         padding-bottom: 2rem;
+    }
+    
+    .installation {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: flex-start;
+        justify-content: space-between;
+    }
         
-        .installation {
-            display: flex;
-            flex-wrap: wrap;
-            align-items: center;
-            justify-content: space-between;
-        }
-        
-        .installation .column {
-            max-width: $cols4;
-        }
+    .installation-column {
+        max-width: $cols4;
+    }
+    
+    .installation-downloads {
+        list-style: none;
+    }
+    
+    .installation-downloads-item {
+        padding: 0;
+        margin: 0.25rem;
```

Comment:
@TheGrits So close! You ran afoul of the strict linter rules. Change the order (margin, then padding), and leave off the leading zero, then this should be ready.